tests/asyncio/test_grpc_helpers_async.py

```diff
@@ -298,48 +298,82 @@ def test_wrap_errors_streaming(wrap_stream_errors):
     wrap_stream_errors.assert_called_once_with(callable_)
 
 
-@mock.patch("grpc.composite_channel_credentials")
+@pytest.mark.parametrize(
```

Comment:
Same comments as in sync version apply here.

Reply:
Done in https://github.com/googleapis/python-api-core/pull/583/commits/396d8fa364daacbcfa5eafcb2b6c51f02504995a